src/main/java/org/cactoos/scalar/ItemAt.java

```diff
@@ -169,7 +169,7 @@ public ItemAt(final Iterable<T> iterable, final int position) {
      * @param iterator Iterator
      * @param position Position
      * @param fallback Fallback value
-     * @todo Remove iterator constructor from org.cactoos.scalar.ItemAt
+     * @todo #1094 Remove iterator constructor from org.cactoos.scalar.ItemAt
```

Comment:
@mazdack this is still not how this work, please do watch the video and look at other todos in the code to see how it should be. The issue referenced should be the one you are currently solving for example. Also be sure the description is long enough, if not pdd won't be happy.